docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
Okie, I'll fixed for this PR. But I'll still think that undefined is best practice for this case.

1. `undefined` with great with `propTypes`, especially with `isRequired` option.
2. if the property change from `undefined` to `null`, the `componentWillReceiveProps` would have capable of noticing difference.
3. When using with ES7, for example, in `render()`
```
const { open = this.state.open } = this.props
```

If we pass `null`, the above code won't work. `undefined` works otherwise.